examples/rstudio/rstudio_auth

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
Why use `bash` from the path instead of the usual `#!/bin/bash`?

Reply:
All the issues with this file are related to me not altering it from the source. :)

Reply:
OK, what is the source? Are we able to re-distribute it? This information should be commented.